### DISCUSSION TITLE
Working on MFA

### DISCUSSION THREAD

**Discussion by MathisMallet** [MaxHalford/prince#172] [General]
Hi, I'm currently working with MFA and I wondered if you would be interested in completing the one you propose as it is missing many features. If so I could try to implement them consistently with your code as I'm already doing it inconsistently.

**Comment by MaxHalford** [comment]
Hey there Mathis!

What exactly do you mean by "the one you propose"?

I'd gladly welcome contributions to the MFA class. What features did you implement on your side?

PS: we can discuss in French if you're more comfortable with that 🥖
